broker-daemon/block-order-worker/index.js

```diff
@@ -847,6 +847,38 @@ class BlockOrderWorker extends EventEmitter {
     })
   }
 
+  /**
+   * Get all completed trade info
+   * @returns {Object} result
+   * @returns {Array<Object>} result.orders all orders for a user
+   * @returns {Array<Object>} result.fills all fills for a user
+   */
+  async getTrades () {
```

Comment:
It's weird to name this `getTrades` if it retrieves all orders and fills, not just wants that are "trades" (i.e. executed). Why not filter inside of this function?

Reply:
Good call, I'll make that change.

Reply:
@martineehrlich ^^